enhancements/ingress/set-delete-http-headers.md

```diff
@@ -732,32 +732,51 @@ We don't allow setting the `strict-transport-security` header on either an Ingre
 #### X-Forwarded-For
 `x-forwarded-for` and related headers were considered for the deny-list (as modifying these headers using this new API could affect configuration specified using
 another [option](https://docs.openshift.com/container-platform/4.12/networking/ingress-operator.html#nw-using-ingress-forwarded_configuring-ingress))
-but have actual use-cases (which aren't covered by the existing option). They will be allowed, with notes in the API and product documentation,
-and possibly a warning on admission.
+but have actual use-cases (which aren't covered by the existing option). Therefore, they will be allowed, with the following note in the product documentation:
 
-For example: If someone creates an Ingress Controller with the following:
-```sh
-apiVersion: operator.openshift.io/v1
-kind: IngressController
-metadata:
-  name: default
-  namespace: openshift-ingress-operator
-spec:
-  httpHeaders:
-    forwardedHeaderPolicy: Replace
-```
-Or if someone add a route annotation `haproxy.router.openshift.io/set-forwarded-headers`,
-then the `set-header` stanzas will be added in `haproxy.config` using the [haproxy-config.template](https://github.com/openshift/router/blob/0cd9a58dff77127562f7851ccfa5cda79b8cef48/images/router/haproxy/conf/haproxy-config.template#L565-L580)
+Note: In the IngressController, if both the action `Set` is configured using `spec.httpHeaders.actions` and the 
+action `Replace` is configured using `spec.httpHeaders.forwardedHeaderPolicy`, then the resulting configured value
+is from `spec.httpHeaders.actions`.  
+In short, the value set by `spec.httpHeaders.actions` will override the value set by `spec.httpHeaders.forwardedHeaderPolicy`.
+
+In the case of HTTP request headers, the actions specified in `spec.httpHeaders.actions` on the Route will be executed after
+the actions specified in the IngressController's `spec.httpHeaders.actions` field.
+
+This enhancement provides a method for a user to  delete the `x-forwarded-for` header, which the Ingress Controller `spec.httpHeaders.forwardedHeaderPolicy` does not provide.
```

Comment:
I would add this to the Note as well (rewording it to remove the reference to an enhancement).